Core/src/org/sleuthkit/autopsy/communications/AccountNode.java

```diff
@@ -88,20 +93,32 @@ protected Sheet createSheet() {
             s.put(properties);
         }
 
+        // RAMAN TBD: need to figure out how to get the right filters here
+        // We talked about either creating a wrapper class around AccountDeviceInstance to push the selected filters
+        // Or some kind of static access to pull the selected filters
+        long msgCount = 0;
+        try {
+            CommunicationsFilter filter = null;
+            msgCount = Case.getCurrentCase().getSleuthkitCase().getCommunicationsManager().getRelationshipsCount(filter, accountDeviceInstance);  
+        }
+        catch (TskCoreException ex) {
+            Exceptions.printStackTrace(ex);
```

Comment:
Please propagate exceptions or log them instead of writing stack traces to stdout. Don't forget to remove the import of Exceptions when you change this.